src/app/query-runner.service.ts

```diff
@@ -161,12 +160,26 @@ export class QueryRunnerService {
     query.duration = (new Date()).getTime() - query.requestSentAt.getTime();
     query.statusCode = res.status;
 
-    AppComponent.messageBarContent = {
+    AppComponent.requestStatus = {
       text: this.createTextSummary(query),
       backgroundClass: this.isSuccessful(query) ? 'ms-MessageBar--success' : 'ms-MessageBar--error',
       icon: this.isSuccessful(query) ? 'ms-Icon--Completed' : 'ms-Icon--ErrorBadge',
     };
 
+    const queries = [
+      'https://graph.microsoft.com/v1.0/me/',
+      'https://graph.microsoft.com/v1.0/user/',
+    ];
+
+    const hasQuery = queries[0].match(query.requestUrl) ||
+      queries[1].match(query.requestUrl);
+
+    const isGET = query.method === 'GET';
+
+    if (this.isSuccessful(query) && hasQuery && isGET) {
+      this.displayTipMessage('Use $select for additional properties');
```

Comment:
Not sure if this is possible, but can we put a link to either the generic query parameters topic OR to the resource topic that shows all the properties (like https://docs.microsoft.com/graph/api/resources/user?view=graph-rest-1.0).

Reply:
This is a good idea, Dan. 

I'd like for us to take a step back here and not hard code these messages. This is a good opportunity to provide contextual information based on the call being exercised. Much like we have the samples, I'd like to see a resource file keyed so that we can match resources, http methods, and query params to a message to be shown in the tips. 

```
if (this.isSuccessful(query)) {
   const tipMessage = this.getTipMessage(query); // Does a look up based resource, method, and query
   this.displayTipMessage(tipMessage);
}
```